scala/scala.bzl

```diff
@@ -349,13 +355,17 @@ def scala_repositories(scala_version = "2.11.11"):
       actual = "@scalac_rules_commons_io//jar")
 
   native.bind(
-      name = "io_bazel_rules_scala/dependency/scalatest/scalatest",
+      name = "io_bazel_rules_scala/dependency/scalatest/scalatest_library",
       actual = "@scalatest_{}//jar".format(major_version_underscore))
 
   native.bind(
       name = "io_bazel_rules_scala/dependency/scalactic/scalactic",
       actual = "@scalactic_{}//jar".format(major_version_underscore))
 
+  native.bind(
```

Comment:
I think you only need this bind. Why do you need to bind scalactic and scalatest_library as well?

Reply:
`scala/scalatest:scalatest` needs to export the correct versions of scalatest/scalactic and here the binds are used to select which versions it exports. I think I'll do it with an if statement instead, that's probably clearer